desktop/cmp/grouping/GroupingChooser.ts

```diff
@@ -91,14 +96,21 @@ export const [GroupingChooser, groupingChooser] = hoistCmp.withFactory<GroupingC
                         ),
                         minimal: styleButtonAsInput,
                         ...buttonProps,
-                        onClick: () => model.toggleEditor()
+                        onClick: () => model.toggleEditor(),
+                        testId
                     }),
-                    favoritesIcon()
+                    favoritesIcon({testId: favoritesIconTestId})
```

Comment:
Looks like currently `favoritesIcon` is dropping the `testId` prop getting passed here

Reply:
@Ryanseanlee Wanted to make sure you saw this- everything else looks good!